src/solver/utils/ortools_utils.cpp

```diff
@@ -0,0 +1,481 @@
+
+void transferVariables(operations_research::MPSolver * solver, double * bMin, double * bMax, double * costs, int nbVar) {
+	MPObjective* const objective = solver->MutableObjective();
+	for (int idxVar = 0; idxVar < nbVar; ++idxVar) {
+		std::ostringstream oss;
+		oss << "x" << idxVar;
+		double min_l = 0.0;
+		if(bMin != NULL) {
+			min_l = bMin[idxVar];
+		}
+		double max_l = bMax[idxVar];
+		auto x = solver->MakeNumVar(min_l, max_l, oss.str());
+		objective->SetCoefficient(x, costs[idxVar]);
+	}
+}
+
+void transferRows(operations_research::MPSolver * solver, double * rhs, char * sens, int nbRow) {
+	for (int idxRow = 0; idxRow < nbRow; ++idxRow) {
+		double bMin = -MPSolver::infinity(), bMax = MPSolver::infinity();
+		if (sens[idxRow] == '=') {
+			bMin = bMax = rhs[idxRow];
+		}
+		else if (sens[idxRow] == '<') {
+			bMax= rhs[idxRow];
+		}
+		else if (sens[idxRow] == '>') {
+			bMin = rhs[idxRow];
+		}
+		std::ostringstream oss;
+		oss << "c" << idxRow;
+		MPConstraint* const ct = solver->MakeRowConstraint(bMin, bMax, oss.str());
+	}
+}
+
+void transferMatrix(operations_research::MPSolver * solver, int * indexRows, int * terms, int * indexCols, double * coeffs, int nbRow) {
+	auto variables = solver->variables();
+	auto constraints = solver->constraints();
+
+	for (int idxRow = 0; idxRow < nbRow; ++idxRow) {
+		MPConstraint* const ct = constraints[idxRow];
+		int debutLigne = indexRows[idxRow];
+		for (int idxCoef = 0; idxCoef < terms[idxRow]; ++idxCoef)
+		{
+			int pos = debutLigne + idxCoef;
+			ct->SetCoefficient(variables[indexCols[pos]], coeffs[pos]);
+		}
+	}
+}
+
+MPSolver * convert_to_MPSolver(PROBLEME_SIMPLEXE * problemeSimplexe) {
+
+    //Define solver used depending on global solver option
+    MPSolver::OptimizationProblemType solverType = Antares::Data::OrtoolsUtils().getLinearOptimProblemType(Antares::Data::OrtoolsUtils::OrtoolsEnumUsed);
+
+	// Create the linear solver instance
+	MPSolver * solver = new MPSolver("simple_lp_program", solverType);
+
+	// Create the variables and set objective cost.
+	transferVariables(solver, problemeSimplexe->Xmin, problemeSimplexe->Xmax, problemeSimplexe->CoutLineaire,  problemeSimplexe->NombreDeVariables);
+
+	// Create constraints and set coefs
+	transferRows(solver, problemeSimplexe->SecondMembre, problemeSimplexe->Sens, problemeSimplexe->NombreDeContraintes);
+	transferMatrix(solver, problemeSimplexe->IndicesDebutDeLigne, problemeSimplexe->NombreDeTermesDesLignes, problemeSimplexe->IndicesColonnes, problemeSimplexe->CoefficientsDeLaMatriceDesContraintes, problemeSimplexe->NombreDeContraintes);
+
+	return solver;
+}
+
+MPSolver * convert_to_MPSolver(PROBLEME_A_RESOUDRE * problemeAResoudre) {
+
+    //Define solver used depending on global solver option
+    MPSolver::OptimizationProblemType solverType = Antares::Data::OrtoolsUtils().getMixedIntegerOptimProblemType(Antares::Data::OrtoolsUtils::OrtoolsEnumUsed);

```

Comment:
For Probleme A resoudre we use Mixed integer optim problem.
This was PNE before.